test/fixtures/pvc/pod.yml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 kind: Pod
 metadata:
   annotations:
-    kubernetes-deploy.shopify.io/timeout-override: 20s
+    kubernetes-deploy.shopify.io/timeout-override: 10s
```

Comment:
Why this number in particular? What could go wrong if we set the timeout very low?

Reply:
Pod needs less than 2 seconds if we drop the bad PVC, so I'm not that worried about the timeout being caused by the pod actually needing more time to complete as opposed the real issue we are testing.